### ISSUE TITLE
user.name return null with Google oauth 2.0

### ISSUE THREAD

**Issue by cescoferraro** [authomatic/authomatic#90]
all other proprieties do too! I've teste with locale, link and so on.
I can log though. Facebook works perfect! 

'google': {
        'class_': oauth2.Google,
        'consumer_key': '###',
        'consumer_secret': '###',
        'id': authomatic.provider_id(),
        'scope': oauth2.Google.user_info_scope ,
    },

**Comment by peterhudec** [comment]
Could you please provide more information (code)? 
Tests for `oauth2.Google` are passing without any errors.

**Comment by cescoferraro** [comment]
i am using the flask example on the werkzeug_adapter folder.
All I've done was insert the snippet above to the config.py and
`Login with <a href="login/google">Google</a>.<br/>` to the index.html to make things function.
I dont get Fb status as well, but it might be my poor config on the developer.FB dashboard.
While debbugin I have set a Pusher alert like this
`p['test_channel'].trigger('my_event', {'message': result.user.name })`
      ` result.user.update()`
       `p['test_channel'].trigger('my_event', {'message': result.user.name }) `
Before user.update() a get null on both fb and google, but google keeps null even after the update()

**Comment by xch91** [comment]
I believe I'm facing the same issue here. The result object from calling login method of Google provider returns an error:

```
{
  "error" : "invalid_request",
  "error_description" : "OAuth 2 parameters can only have a single value: client_secret"
}
```

I don't think this can be fixed without modifying the code XD

This link might provide some insight I guess: http://www.cloudsponge.com/blog//stories/2015/04/14/gmail-oauth2-broken-and-fixed.html

**Comment by xch91** [comment]
Just found that this bug has been fixed in commit b3c9ade80f43ae020027a551935b565fa133a454

**Comment by cescoferraro** [comment]
I did not reply cause it was not an error with code, all I had to do was add the Google+ credential on he console.developer.google.com as answered on stackoverflow. I think this should be mentioned on the docs because once it is not obvious to figure out.

**Comment by datashed-lewis** [comment]
I believe the original post issue is the same as:
http://stackoverflow.com/questions/27196887/google-and-twiter-return-empty-user-data-on-flask

You actually need to enable to the Google+  api on the google developer dashboard.

**Comment by cescoferraro** [comment]
@datashed-lewis that was it, but at the same time i have had the same issue fixed on the b3c9ade
